volatility/framework/plugins/windows/envars.py

```diff
@@ -0,0 +1,159 @@
+# This file is Copyright 2020 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+# Creator: Aviel Zohar (memoryforensicsanalysis@gmail.com) # Fix vol2 plugin to vol3

```

Comment:
I just noticed this, and to avoid people getting territorial, we'd prefer that source code not contain contributor names.  The git history should tell people who authored this, and we're happy to have your name appear in a CREDITS file, but we're prefer not to have names directly in the codebase if possible?  5:S